source/braille.py

```diff
@@ -1605,6 +1624,14 @@ def setTether(self, tether, auto=False):
 	def _get_shouldAutoTether(self):
 		return self.enabled and config.conf["braille"]["autoTether"]
 
+	def _get_displaySize(self):

```

Comment:
Note that this is at least set by the alva driver in core when changing the split point, and then it will break this. You will have to change the alva driver to use braille.handler._displaySize instead.